### DISCUSSION TITLE
Parsing Json array values

### DISCUSSION THREAD

**Discussion by rishyank** [grafana/grafana-infinity-datasource#375] [Q&A]
Hi ,How can I split the json array values into 3 different columns using UQL

prase-json
"values":["2022-08-05 14:17:51.389595,23.100000381469727,38.20000076293945","2022-08-05 14:17:52.616600,23.100000381469727,38.20000076293945","2022-08-05 14:17:53.851367,23.100000381469727,38.0","2022-08-05 14:17:55.035585,23.100000381469727,38.0","2022-08-05 14:17:56.300269,23.100000381469727,37.79999923706055","2022-08-05 ]

**Comment by yosiasz** [comment]
This is not a valid json?
